docs/docusaurus/docs/cloud/connect/connect_postgresql.md

```diff
@@ -25,28 +23,51 @@ New to GX Cloud and not sure that it's the right solution for your organization?
 
 - You have stopped all local running instances of the GX Agent.
 
-## Prepare your PostgreSQL environment
+## Connect to a PostgreSQL Data Asset
+
+1. In GX Cloud, click **Data Assets** > **New Data Asset**.
+
+2. Copy the code in the code pane.
+
+3. Prepare your PostgreSQL environment:
+
+   - In pgAdmin, select a database.
+
+   - Click **Tools** > **Query Tool**.
+
+   - Paste the code you copied in step 2 into the **Query** pane to create and assign the `gx_role` role and allow GX Cloud to access to all `public` schemas and tables on a specific database.
+
+      Replace `<your_password>` and `<your_database>` with your own values. `ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT ON TABLES TO gx_role;` is optional and gives the `gx_role` user access to all future tables in the defined schema.
+
+    - Click **Execute/Refresh**.
+
+4.  In GX Cloud, click **I have created a GX Cloud user with valid permissions** and then click **Continue**.
+
+5. Enter a meaningful name for the Data Asset in the **Data Source name** field.
+
+6. Enter a connection string in the **Connection string** field. The connection string format is `postgresql+psycopg2//YourUserName:YourPassword@YourHostname:5432/YourDatabaseName`. 
+
+7. Optional. Select **Test connection** to test the Data Source connection. Testing the connection to the Data Source is a preventative measure that makes sure the connection configuration is correct. This verification can help you avoid errors and can reduce troubleshooting downtime.
+
+8. Click **Continue**.
+
+9. Select **Table Asset** or **Query Asset** and complete the following fields:
 
-1. In pgAdmin, select a database.
+    - **Table name**: When **Table Asset** is selected, enter a name for the table you're creating in the Data Asset.
+    
+    - **Data Asset name**: Enter a name for the Data Asset. Data Asset names must be unique across all Data Sources in GX Cloud.
 
-2. Click **Tools** > **Query Tool**.
+    - **Query**: When **Query Asset** is selected, enter the query that you want to run on the table. 
```

Comment:
I think this is slightly inaccurate.  When query asset is selected, there is no "the table" specified by the user.  Instead, since the query could theoretically join multiple tables, it should be something like:
```suggestion
    - **Query**: When **Query Asset** is selected, enter the query that you want to run on the database.
```

Reply:
@Rachel-Reverie, I attempted to clarify the text.